conditional/blueprints/attendance.py

```diff
@@ -552,28 +552,28 @@ def get_ts_attendees(cid, user_dict=None):
         attendees = [{"value": a.uid,
```

Comment:
and this is cm attendance, (and you didn't change the route name), they're reversed